tutorials/abap-environment-developer-user/abap-environment-developer-user.md

```diff
@@ -1,7 +1,7 @@
 ---
 auto_validation: true
-title: Create a Developer User in SAP Cloud Platform ABAP Environment
-description: Create a developer user with the developer role in SAP Cloud Platform ABAP Environment.
+title: Create a Developer User in SAP BTP ABAP Environment
+description: Create a developer user with the developer role in SAP BTP ABAP Environment.
```

Comment:
Please use the full name "SAP Business Technology Platform" at least once at the beginning of your tutorial before you use the abbreviation.